tests/data_asset/test_data_asset.py

```diff
@@ -147,3 +148,24 @@ def test_valid_expectation_types(dataset, pandas_dataset):
         "expect_table_row_count_to_be_between",
         "expect_table_row_count_to_equal",
     ]
+
```

Comment:
Looking at this -- being `batch_kwargs` focused -- is this PR fixing an issue in `v012`?

Reply:
All the fixes in this PR are for pre-013 API.